index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <link rel="stylesheet" href="style.css">
+
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <header>
+        <section class="toolbar">
+            <div class="tbar">
+                <div class="menu-bar">
+                    <p class="john-doe">IKOOTE</p>
```

Comment:
- For semantic purposes, kindly change this from a `p` tag to an `a` tag. Making the logo a link improves user experience and makes the navigation of the webpage easier. It is also good HTML practice to make the logo of a webpage a link. 👍